gallery_dl/extractor/instagram.py

```diff
@@ -0,0 +1,189 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018 Leonardo Taccari
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 as
+# published by the Free Software Foundation.
+
+"""Extract images from Instagram"""
```

Comment:
The module docstring should end in the full root URL of the site in question.
`Instagram` -> `https://www.instagram.com/`

(This is used to auto-generate `docs/supportedsites.rst`. You should run `./scripts/build_supportedsites.py` and see how this turns out.)

Reply:
I have adjusted them respectively to `image` and and `user` (so
the description in `docs/supportedsites.rst` will also looks nicer
based on the known categories).
I have also renamed the `Instagram*Extractor`s accordingly
(`InstagramPostpageExtractor` -> `InstagramImageExtractor`,
`InstagramProfilepageExtractor` -> `InstagramUserExtractor`). This was
pointed out by `test/test_extractor.py`.